data/data/bootstrap/files/usr/local/bin/installer-gather.sh

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
+
+ARTIFACTS="${1:-/tmp/artifacts}"
+mkdir -p "${ARTIFACTS}"
+
+echo "Gathering bootstrap journals ..."
+mkdir -p "${ARTIFACTS}/bootstrap/journals"
+for service in bootkube openshift kubelet crio
+do
+    journalctl --boot --no-pager --output=short --unit="${service}" > "${ARTIFACTS}/bootstrap/journals/${service}.log"
+done
+
+echo "Gathering bootstrap containers ..."
+mkdir -p "${ARTIFACTS}/bootstrap/containers"
+for container in $(crictl ps --all --quiet)
+do
+    container_name=$(crictl ps -a --id ${container} -v | grep -oP "Name: \K(.*)")
```

Comment:
I would rather use [`--output=json`][1] and then use `python` or some such (we don't have `jq` handy) to extract the container name from the JSON.

[1]: https://github.com/kubernetes-sigs/cri-tools/blob/v1.14.0/cmd/crictl/container.go#L295-L298

Reply:
don't make us depend on python on RHCOS hosts ;)

Reply:
> don't make us depend on python on RHCOS hosts ;)

`grep` for structured data makes me sad ;).  Maybe we can update `crictl` to get Go templating or some such to give us an eventual off ramp?